### ISSUE TITLE
Some strings missing translation

### ISSUE THREAD

**Issue by stevelacerda** [bigbluebutton/bigbluebutton#12340]
I'm using BBB 2.3-dev beta 5 with pt_BR locale.

I saw some missing key translations like _app.actionsBar.actionsDropdown.presentationLabel_ and downloaded the latest pt_BR.json from Transifex (with the missing keys already translated). Even after a complete restart, I keep seeing some untranslated strings, as you can see below.

![temp](https://user-images.githubusercontent.com/5473310/117680502-9ac16900-b187-11eb-87f0-0efc41289d89.png)

Am I missing something?

Thank you.

**Comment by ramonlsouza** [comment]
@stevelacerda we recently moved the locales to another directory, so we can serve them as static assets... Did you copy the pt_BR.json to `/usr/share/meteor/bundle/programs/web.browser/app/locales`?

**Comment by stevelacerda** [comment]
> @stevelacerda locales files were recently moved to another directory, so they can be served as static assets... Did you copy the pt_BR.json to `/usr/share/meteor/bundle/programs/web.browser/app/locales`?

Yes, I did that. I also tried to replace en.json by pt_BR.json and changed the language to english, but some strings are still missing. I checked the key names, but with no luck.

After some digging (while writing this message), I tried to replace some strings in the file to check if it was updated and it started working. Maybe it was a cache issue, but I tried on different browsers and after restarting the services a few times.

Thank you for your time and sorry for the inconvenience. I'll close the issue.
